tools/judge/internal/toyjudge/cli.go

```
@@ -0,0 +1,138 @@
+// 0 通过；2 配置/输入错误（rubric 非法、model.yaml 缺字段、同族 judge 等）；20 κ 门禁未达标。
+const (
+	ExitOK        = 0
+	ExitInput     = 2
+	ExitKappaGate = 20
+)
+
+// Main 是 toyjudge CLI 入口（cmd/toyjudge 为薄壳），返回进程退出码。
+func Main(args []string, stdout, stderr io.Writer) int {
+	const usage = "用法: toyjudge <calibrate|run> [flags]"
+	if len(args) == 0 {
+		fmt.Fprintln(stderr, usage)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)